cl/search/filters.py

```diff
@@ -28,6 +28,10 @@ class CourtFilter(NoEmptyFilterSet):
         "cl.search.filters.DocketFilter", queryset=Docket.objects.all()
     )
     jurisdiction = filters.MultipleChoiceFilter(choices=Court.JURISDICTIONS)
+    parent_court = filters.CharFilter(
+        field_name="parent_court__id",
+        lookup_expr="exact",
```

Comment:
We could simplify the filter addition by omitting the `lookup_expr` argument. The Filter class's [constructor](https://github.com/carltongibson/django-filter/blob/2494df96c6387a9fa411fcb00b696b15dfd9216b/django_filters/filters.py#L75-L103) defaults to using `settings.DEFAULT_LOOKUP_EXPR`, which is typically "exact". Unless we modify the default lookup expression in our settings module, the implicit behavior will suffice.